<ui version="4.0" >
 <class>RepeatEntryBase</class>
 <widget class="QDialog" name="RepeatEntryBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Repeat</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QComboBox" name="repeatTypeSelect" >
     <item>
      <property name="text" >
       <string>Daily</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Weekly</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Monthly</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Yearly</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="FrequencyLabel" >
       <property name="text" >
        <string>Frequency</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="frequency" >
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="dayPage" />
     <widget class="QWidget" name="weekPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Repeat On</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QToolButton" name="weekDay1Toggle" >
              <property name="text" >
               <string comment="One letter for Monday" >M</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="weekDay2Toggle" >
              <property name="text" >
               <string comment="One letter for Tuesday" >T</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="weekDay3Toggle" >
              <property name="text" >
               <string comment="One letter for Wednesday" >W</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="weekDay4Toggle" >
              <property name="text" >
               <string comment="One letter for Thursday" >T</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="weekDay5Toggle" >
              <property name="text" >
               <string comment="One letter for Friday" >F</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QToolButton" name="weekDay6Toggle" >
              <property name="text" >
               <string comment="One letter for Saturday" >S</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="weekDay7Toggle" >
              <property name="text" >
               <string comment="One letter for Sunday" >S</string>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="monthPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="monthSubtypeSelect" >
         <property name="title" >
          <string>Repeat On</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>5</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="dayOfMonth" >
            <property name="text" >
             <string>day of month</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="weekDayOfMonth" >
            <property name="text" >
             <string>day of week from
start of month</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lastWeekDayOfMonth" >
            <property name="text" >
             <string>day of week from
end of month</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="yearPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>repeatTypeSelect</tabstop>
  <tabstop>frequency</tabstop>
  <tabstop>weekDay1Toggle</tabstop>
  <tabstop>weekDay2Toggle</tabstop>
  <tabstop>weekDay3Toggle</tabstop>
  <tabstop>weekDay4Toggle</tabstop>
  <tabstop>weekDay5Toggle</tabstop>
  <tabstop>weekDay6Toggle</tabstop>
  <tabstop>weekDay7Toggle</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>repeatTypeSelect</sender>
   <signal>activated(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>85</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel" >
     <x>102</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
